command/task_create.go

```diff
@@ -274,9 +274,18 @@ func handleDeprecations(ui mcli.Ui, tc *config.TaskConfig) error {
 				action := generateServiceModuleInputBlockAction(tc.Services)
 				ui.Output(generateServiceFieldMsg(action))
 			case *config.ScheduleConditionConfig:
-				action := generateServicesAction(tc.Services)
+				action := generateServiceModuleInputBlockAction(tc.Services)
 				ui.Output(generateServiceFieldMsg(action))
```

Comment:
It looks like all the other conditions output the same message. Would it be possible to consolidate this by replacing with a `default` block? Maybe there's something I'm not thinking of?

Reply:
yes, you're right, they're all the same now. Good point.